### PR TITLE
[16.0][MIG] stock_landed_costs_security: Migration to version 16.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
-Standard migration to v16

@Tecnativa
TT47864

@pilarvargas-tecnativa @chienandalu

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_landed_costs_security/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#. Go to Users.
+#. Select Landing costs -> User.
```
Comment (pilarvargas-tecnativa) [inline stock_landed_costs_security/readme/USAGE.rst:2]:
```suggestion

```
This is explained in CONFIGURE.rst, isn't it?

Comment (carolinafernandez-tecnativa) [reply stock_landed_costs_security/readme/USAGE.rst:2]: In CONFIGURE.rst, it explains how configure the module and in this file how to use it, i think its okay this file with this information

Comment (pedrobaeza) [reply stock_landed_costs_security/readme/USAGE.rst:2]: cc @carolinafernandez-tecnativa

File: stock_landed_costs_security/views/stock_landed_costs_view.xml [hunk 2]
```diff
@@ -13,12 +13,6 @@
             </button>
         </field>
     </record>
-    <record id="stock_landed_costs.account_view_move_form_inherited" model="ir.ui.view">
-        <field
-            name="groups_id"
-            eval="[(4, ref('stock_landed_costs_security.group_stock_landed_costs_security_user'))]"
```
Comment (pedrobaeza) [inline stock_landed_costs_security/views/stock_landed_costs_view.xml]: You must put this group on each of the fields of the view.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: stock_landed_costs_security/views/stock_landed_costs_view.xml (hunk 2 above)
Comment (carolinafernandez-tecnativa) [reply stock_landed_costs_security/views/stock_landed_costs_view.xml]: I have checked and the fields that are invisibile should i add the group also? https://github.com/odoo/odoo/blob/16.0/addons/stock_landed_costs/views/account_move_views.xml#L3
I tried on my local but adding the groups in the others fields that are invisible i got error message.

Comment (pedrobaeza) [reply stock_landed_costs_security/views/stock_landed_costs_view.xml]: As said, you now need to put the groups attribute on each of the elements of the view:

image

Comment (carolinafernandez-tecnativa) [reply stock_landed_costs_security/views/stock_landed_costs_view.xml]: Done! Thanks

File: stock_landed_costs_security/views/stock_landed_costs_view.xml [hunk 3]
```diff
@@ -39,6 +33,46 @@
                     separator=","
                 />
             </button>
+            <xpath
+                expr="//field[@name='invoice_line_ids']/tree/field[@name='product_type']"
```
Comment (pedrobaeza) [inline stock_landed_costs_security/views/stock_landed_costs_view.xml]: This one is not needed. It's an invisible field.

File: stock_landed_costs_security/views/stock_landed_costs_view.xml [hunk 4]
```diff
@@ -39,6 +33,46 @@
...
+                    separator=","
+                />
+            </xpath>
+            <xpath
+                expr="//field[@name='invoice_line_ids']/tree/field[@name='is_landed_costs_line']"
+                position="attributes"
+            >
+                <attribute
+                    name="groups"
+                    add="stock_landed_costs_security.group_stock_landed_costs_security_user"
```
Comment (pedrobaeza) [inline stock_landed_costs_security/views/stock_landed_costs_view.xml]: This syntax is not valid for these views. It's only for QWeb templates.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**